### DISCUSSION TITLE
Not able to get round value of 1 in y_pred_probs tensor in Part 70

### DISCUSSION THREAD

**Discussion by jeremyitty** [mrdbourke/pytorch-deep-learning#1337] [Q&A]
@ 10:21:30 hrs

While using sigmoid function for y_logits while defining y_pred_probs function, not able to get values close to 1.
Still showing values of 0 while using `torch.round`.

Tried rerunning and restarting, but still issue wont fix.
Please have a look and offer suggestions !

<img width="1086" height="652" alt="Screenshot 2026-04-05 204630" src="https://github.com/user-attachments/assets/f8a77ba7-2a5c-42ee-a2e2-878f7adce69b" />
